miniapps/nurbs/ex1p.cpp

```diff
@@ -1,24 +1,24 @@
 //                     MFEM Example 1 - Parallel NURBS Version
 //
-// Compile with: make ex1p
```

Comment:
Not quite sure why you changed this name? this should stay ex1p.

Reply:
Cmake requires unique executable names.